addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+
+   > Note: The `<REPOSITORY>` must be a location in the docker registry that can be written to with the credentials
+   > - For [dockerhub](https://hub.docker.com/) this should be `my-username/my-repo`.
+   > - For [GCR](https://cloud.google.com/container-registry/) this should be `gcr.io/my-project/my-repo`.
+
+2. Create a cluster store
+
+   A store resource is a repository of [buildpacks](http://buildpacks.io/) packaged
+   in [buildpackages](https://buildpacks.io/docs/buildpack-author-guide/package-a-buildpack/) that can be used by kpack
+   to build OCI images. Later in this tutorial, you will reference this store in a Builder configuration.
+
+   We recommend starting with buildpacks from the [paketo project](https://github.com/paketo-buildpacks). The example
+   below pulls in java buildpack from the paketo project. If you would like to use a different language you can select a different buildpack.
+
+    ```
+    kp clusterstore save default -b gcr.io/paketo-buildpacks/java
+    ```
+
+   > Note: Buildpacks are packaged and distributed as buildpackages which are OCI images available on a docker registry. Buildpackages for other languages are available from [paketo](https://github.com/paketo-buildpacks).
+
+3. Create a cluster stack
+
+   A stack resource is the specification for
+   a [cloud native buildpacks stack](https://buildpacks.io/docs/concepts/components/stack/) used during build and in the
+   resulting app image.
+
+   We recommend starting with the [paketo base stack](https://github.com/paketo-buildpacks/stacks) as shown below:
+
+    ```
+    kp clusterstack save base --build-image paketobuildpacks/build:base-cnb --run-image paketobuildpacks/run:base-cnb
+    ```
+
+4. Create a Builder
+
+   A Builder is the kpack configuration for a [builder image](https://buildpacks.io/docs/concepts/components/builder/)
+   that includes the stack and buildpacks needed to build an OCI image from your app source code.
+
+   The Builder configuration will write to the registry with the secret configured in step one and will reference the
+   stack and store created in step three and four. The builder order will determine the order in which buildpacks are
+   used in the builder.
+
+    ```
+    kp builder save my-builder \
+      --tag <IMAGE-TAG> \
+      --stack base \
+      --store default \
+      --buildpack paketo-buildpacks/java \
+      -n default
+    ```
+
+    - Replace `<IMAGE-TAG>` with a valid image tag that exists in the registry you configured with the `--registry` flag when creating a Secret in step #1. The tag should be something like: `your-name/builder` or `gcr.io/your-project/builder`
```

Comment:
I don't see the --registry flag mentioned in step 1?

Reply:
I will remove that